gemini/prompts/examples/text_extraction.ipynb

```diff
@@ -0,0 +1,630 @@
+{
```

Comment:
<p>Change this to just be "Vertex AI"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/478/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>